spec/support/data_uri_helper.rb

```diff
@@ -0,0 +1,13 @@
+DataUri = Struct.new(:media_type, :data) do
+  REGULAR_EXPRESSION = %r{^data:(?<media_type>[\w/\-.;=]+?)(?<base64>base64){0,1},(?<data>.*)}
+  def self.parse(data_uri)
+    matches = REGULAR_EXPRESSION.match(data_uri)
+    media_type = matches[:media_type]&.split(';')
+    data = if matches[:base64].present?
+             Base64.decode64(match[:data])
+           else
+             URI.decode_uri_component(matches[:data])
+           end
+    self.new(media_type, data)
+  end
+end
```

Comment:
maybe we just ditch this and use our own class? the reason I added the `data-uri` gem was to try to have outside parsing of `data://` URIs but maybe we just reuse our own class `Idv::DataUriImage` in the two specs we used that in?

Reply:
Yeah, I think the `DataUrlImage` class could probably do the work here. Maybe the class should be renamed, since it's neither specific to images nor IdV?

Reply:
Thanks, I had missed that we had a class already, will use that!

Reply:
The PR to fix it has been open for a year, so I'm not super optimistic: https://github.com/dball/data_uri/pull/10